crates/tuono/Cargo.toml

```diff
@@ -30,6 +30,7 @@ reqwest = { version = "0.12.4", features = ["blocking", "json"] }
 serde_json = "1.0"
 fs_extra = "1.3.0"
 http = "1.1.0"
+wiremock = "0.6.2"
```

Comment:
This is used only for tests, so I think it should go into `dev-dependencies`: https://docs.rs/wiremock/latest/wiremock/